### DISCUSSION TITLE
Only receiving geo+json content-type back from the API

### DISCUSSION THREAD

**Discussion by telangn** [weather-gov/api#247] [Q&A]
I am making a GET request with the 'Accept' header asking for 'application/cap+xml' back from the API (and any other format as well) but only geo+json is returned back every time. No other format is returned back no matter which one I ask for.

```
import * as chai from 'chai';
import chaiHttp = require('chai-http');
import chaiAsPromised = require('chai-as-promised');
import 'mocha';

chai.use(chaiHttp);
chai.use(chaiAsPromised);
const expect = chai.expect;
const url = 'https://api.weather.gov';

describe('Weather API', async () => {

    it('Should be up and running', async () => {
        var response = await chai.request(url).get('/');
        return expect(response).to.be.status(200);
    });

    it('Should return weather of washington monument', async () => {
        var response = await chai.request(url).get('/gridpoints/LWX/96,70/forecast')
            .set('User-Agent', 'test@gmail.com')
            .set({Accept: 'application/cap+xml'});  // I have also tried .set('Accept', 'application/cap+xml');
        console.log(response.body.properties.periods);
        console.log(response.status);
        console.log(response.header); // RESPONSE IS ALWAYS geo+json content-type
        return expect(response).to.be.status(200);
    });
});
```

**Comment by JJKraw** [comment]
Go to https://www.weather.gov/documentation/services-web-api, click on the Specification tab, and then scroll down and expand the entry for "/gridpoints/{wfo}/{x},{y}/forecast". Once that is open, scroll to Responses and look at the pulldown below 200 / success. That enumerates the formats supported for that endpoint. "cap+xml" is not supported.

I know cap+xml is supported for /alerts/{id}. I'm not sure about any others. It doesn't really make much sense for anything other than alerts since it is a format for alerts.

**Comment by telangn** [comment]
@JJKraw Thank you! Based on that endpoint, Ld+json is now working for me, but I get a 500 for application/vnd.noaa.dwml+xml (which is a valid Accept based on the specification).

Same result if I try it out on the website as well.  Is it a bug ?

**Comment by scadergit** [comment]
@telangn please paste the full error message so we can help further.

**Comment by JJKraw** [comment]
This is from the API spec I mentioned above, using the "Try It Out" feature:

(hope the screen shot paste works...)

![Screen Shot 2021-01-24 at 9 43 57 ](https://user-images.githubusercontent.com/67399409/105633830-ee05ea80-5e28-11eb-9d65-6a9088645f6e.png)

**Comment by telangn** [comment]
Yes, this is what I received as well, from my testing. Same info came back. Is it a bug ?

**Comment by scadergit** [comment]
It appears that NCO is still recovering from their datacenter-wide outage. This impacts the API's ability to pull all the parts together necessary to produce the response. You can email the helpdesk and ask to be notified when the issue is resolved.
